README.md

```diff
@@ -15,8 +15,35 @@
 
 # <a name="overview"></a>Overview
```

Comment:
Do you know what is creating those links? I don’t think they’re useful—or syntactically correct for that matter. Maybe a VS code extension?

Reply:
I am creating these links :)

As far as I know, this is the only way to support linking to parts of a markdown doc in GH flavored markdown. It is not syntactically correct HTML, but it works for this purpose.

Reply:
I think GitHub does this automatically for headers! If you look at [this README](https://github.com/orbit-love/notion-ruby-client/blob/main/README.md), you can click on the ToC to land on a specific header, but I didn’t add any link/attribute to the markdown there

Reply:
And if you’d like to add an anchor to something that’s not a header, then I think you need to use the `id` attribute, the `name` appears to be deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#deprecated_attributes